.github/workflows/copilot_deploy.yml

```diff
@@ -18,7 +18,7 @@ on:
         description: Run performance tests
```

Comment:
Can you add the §run-name§ bit to show which envs are being deployed to please? (underneath name but it won't let me comment there)
`run-name: AWS Deploy ${{ github.event_name == 'workflow_dispatch' && github.event.inputs.environment || (github.ref == 'refs/heads/main' && 'Dev-Test-UAT-Prod' || 'Dev-Test') }}`

Reply:
Added.